playground_test.go

```diff
@@ -2383,6 +2383,34 @@ func TestScriptExecutions(t *testing.T) {
 		require.Empty(t, resp.CreateScriptExecution.Errors)
 	})
 
+	t.Run("Multiple translation (missing imports)", func(t *testing.T) {
```

Comment:
Could you add in adapters/address_test a test that contains a script with import addresses and return value, and test that it does it correctly in each direction. There's no test that checks that unfortunately and we should improve this as well.

Reply:
gj on this test

Reply:
Added a test :)